paddle/fluid/operators/distributed/grpc_sendrecvop_utils.cc

```diff
@@ -0,0 +1,157 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
`utils` can not indicate what this file is doing, why not rename it to `grpc_serde.cc`?

Reply:
Done.